src/Commands/core/CacheCommands.php

```diff
@@ -286,8 +287,14 @@ public function getTypes($include_bootstrapped_types = false)
      */
     public static function clearDrush()
     {
-        drush_cache_clear_all(null, 'default'); // commandfiles, etc.
-        drush_cache_clear_all(null, 'factory'); // command info from annotated-command library
+        try {
+            drush_cache_clear_all(null, 'default');// commandfiles, etc.
+            drush_cache_clear_all(null, 'factory'); // command info from annotated-command library
+        } catch (IOException $e) {
+            // Sometimes another process writes files into a bin dir and \Drush\Cache\FileCache::clear fails.
+            // That is not considered an error.
+            Drush::logger()->info($e->getMessage());
```

Comment:
maybe better to `logger->error()` instead of info?

Reply:
or warning to return exit code 0

Reply:
We usually throw exceptions rather than log to error(), but maybe warning would be appropriate if we go this route.